Content.Shared/_Impstation/Construction/Steps/RemoveEntityConstructionGraphStep.cs

```diff
@@ -0,0 +1,45 @@
+using Content.Shared.Construction;
+using Content.Shared.Construction.Steps;
+using Content.Shared.Examine;
+using Content.Shared.Tag;
+
+
+namespace Content.Shared._Impstation.Construction.Steps
+{
+    [DataDefinition]
+    public sealed partial class EntityRemoveConstructionGraphStep : ConstructionGraphStep
+    {
+        /// <summary>
+        /// A tag of the item you want to remove.
+        /// </summary>
+        [DataField("remove")]
+        private string? _tag;
+        [DataField("name")] public string Name { get; private set; } = string.Empty;
```

Comment:
attributes (`DataField`) should be on their own line, like seen above with `_tag`. these should have spacing between them too, everything's a little cluttered